[log]
level = "ERROR"

[providers]
[providers.docker]
exposedByDefault = false
endpoint = "unix:///var/run/docker.sock"
network = "traefik_proxy"
[providers.file]
filename = "/etc/traefik/dynamic.yml"

[api]
dashboard = false

# https://doc.traefik.io/traefik/routing/entrypoints/#redirection
[entryPoints.web]
address = ":80"

[entryPoints.web.http]
[entryPoints.web.http.redirections]
[entryPoints.web.http.redirections.entryPoint]
to = "websecure"
scheme = "https"

[entryPoints.websecure]
address = ":443"

[certificatesResolvers]
[certificatesResolvers.myresolver.acme]
email = "user@domain.local" # change this email in production
storage = "/letsencrypt/acme.json"
[certificatesResolvers.myresolver.acme.tlsChallenge]
